src/Scouting.elm

```diff
@@ -0,0 +1,141 @@
+module Scouting exposing (main)
+
+import Browser
+import Html
+import Html.Attributes exposing (placeholder, style, value)
+import Html.Events exposing (onClick, onInput)
+import Maybe
+import String
+
+
+main : Program () Model Msg
+main =
+    Browser.sandbox
+        { init = init
+        , view = view
+        , update = update
+        }
+
+
+type Msg
+    = NameInput String
+    | TeamInput String
+    | MatchInput String
+    | DriverStationInput String
+    | Start
+
+
+type alias Model =
+    { scouterName : String
+    , teamNum : Int
+    , matchNum : Int
+    , driverStation : String
+    , isStarted : Bool
+    }
+
+
+init : Model
+init =
+    { scouterName = ""
+    , teamNum = 0
+    , matchNum = 0
+    , driverStation = ""
+    , isStarted = False
+    }
+
+
+dropDown_driverStation : String
+dropDown_driverStation =
+    "visible"
```

Comment:
The conventional name is `dropDownDriverStation`.
Please adhere to `camelCase` conventions.